translations/ui-orders/en.json

```diff
@@ -737,7 +723,7 @@
   "orderLineList.poLineNumber": "POL number",
   "orderLineList.updatedDate": "Updated date",
   "orderLineList.title": "Title",
-  "orderLineList.productIds": "Product ID",
+  "orderLineList.productIds": "Product id",
```

Comment:
are you sure about this? Usually `ID` is a short form of identifier
https://github.com/folio-org/ui-inventory/blob/master/translations/ui-inventory/en.json#L255
https://github.com/folio-org/ui-finc-select/blob/master/translations/ui-finc-select/en.json#L24

Reply:
fixed